aptos-move/aptos-vm/src/aptos_vm_impl.rs

```diff
@@ -173,8 +178,12 @@ impl AptosVMImpl {
         log_context: &AdapterLogSchema,
     ) -> Result<&AptosGasParameters, VMStatus> {
         self.gas_params.as_ref().ok_or_else(|| {
-            log_context.alert();
-            error!(*log_context, "VM Startup Failed. Gas Parameters Not Found");
+            speculative_log!(
+                Level::Error,
+                log_context,
+                String::from("VM Startup Failed. Gas Parameters Not Found"),
```

Comment:
nit:  you could use `into()`, e.g.,`"VM Startup Failed. Gas Parameters Not Found".into()`